<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Klatt</class>
 <widget class="QDialog" name="Klatt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Klatt声码器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Klatt {
border-image: url(:/new/prefix1/background.jpg);
}
#Klatt * {
border-image:url();
}
</string>
  </property>
  <widget class="QPushButton" name="klatt_exit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="klattfigure">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>371</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>122</y>
     <width>101</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="klatt_origin">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>原语音</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="klatt_pulse">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>脉冲激励</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="klatt_EGG">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>EGG 激励</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="klatt_comboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>bF0C11011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bF0C11022</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bF0C11031</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bF0C11041</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bM0C01013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bM0C01023</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bM0C01051</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color:rgba(255,0,0,0);border-radius:0px;}</string>
   </property>
   <property name="text">
    <string>请选择原始语音</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
